### ISSUE TITLE
Design game theme

### ISSUE THREAD

**Issue by labond** [boxart/boxart-boiler#54]
Design game theme.
Coordinate with art vendor to create contracted art for 2 demo games.

**Comment by Wilto** [comment]
Moving this to the next sprint based on our conversations with Billy; he likely won’t have initial sketches ready until the end of the week.

**Comment by labond** [comment]
We have sketches and need to work with Billy to finish design

**Comment by kadamwhite** [comment]
Q: How is billy blocked on his art? A: He has sent us sketches, we need to tell him if he should move ahead, and in what direction.

**Comment by labond** [comment]
Final feedback to Billy 5/13

**Comment by labond** [comment]
Completed feedback cycles  with vendor and received final art 5/26. Art for robot backgrounds and card backings to be commissioned internally starting 5/31; creating a separate ticket for this work.
